content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/150-referential-actions.mdx

```diff
@@ -8,21 +8,23 @@ toc: true
 
 <TopBlock>
 
-The following behavior applies in Prisma versions 2.x.
+Prisma 2 prevents deletion of connected records in some Prisma Client functions, and does not let you configure referential actions in your Prisma Schema to change that behavior.
+  
```

Comment:
It's risky to start using monikers like "Prisma 2" and Prisma 3", without precedent and a clear definition of that those mean. It is much clearer to state "Prisma version 2.x" and "Prisma version 3.x"

Without the word "version" those two monikers can sound like there was some product rebrand or relaunch, and imply that they are different products.

Reply:
Fixed via suggestion in https://github.com/prisma/docs/pull/2784/files#r800954750 maybe?